packages/flutter/test/widgets/editable_text_cursor_test.dart

```diff
@@ -328,6 +328,7 @@ void main() {
     final TextEditingController controller = TextEditingController(text: testText);
     final Widget widget = MaterialApp(
       home: EditableText(
+        key: GlobalKey(),
```

Comment:
Why do we need to add this line? It doesn't look like it's being used?

Reply:
See the discussion at https://github.com/flutter/flutter/pull/75032/files#r570529343

It seems like it's not needed anymore, so I'll remove it and the other places where I added GlobalKeys in the tests.